packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/container.tsx

```diff
@@ -64,7 +63,6 @@ const AccountRecoveryConfirmKeyContainer = (_: RouteComponentProps) => {
       uid
     );
 
-    sensitiveDataClient.setData('reset', { kB });
```

Comment:
Looking at this - I think the intent was to remove `kB` from location state and pass with `sensitiveDataClient` but we may have missed retrieving it from sensitiveDataClient in the `CompleteResetPassword` container. I don't think the duplication with location state was intentional, and we should treat `kB` as sensitive.

Reply:
Thanks for the catch, fixed!